src/js/common/schemaform/definitions/bankAccount.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash/fp';
+import fullSchema1995 from 'vets-json-schema/dist/change-of-program-schema.json';
```

Comment:
This will hopefully change to import a common definitions file, instead of the 1995 schema, since these are common fields and it seems weird for them to depend on a specific form's schema.